pkg/apis/core/types.go

```diff
@@ -3686,6 +3686,7 @@ type NodeSpec struct {
 
 	// Unschedulable controls node schedulability of new pods. By default node is schedulable.
 	// +optional
+	// Deprecated
```

Comment:
normally we mention what the field is deprecated in favor of. this also needs to go on the field under k8s.io/api/... (and generated documentation regenerated) if we want this to be visible to API doc consumers